registry/registry_test.go

```diff
@@ -0,0 +1,317 @@
+package registry
```

Comment:
This file is missing our custom build tag:
```suggestion
// +build !integration

package registry
```